hazelcast/src/main/java/com/hazelcast/config/CacheConfig.java

```diff
@@ -455,6 +457,28 @@ public CacheConfig setQuorumName(String quorumName) {
         return this;
     }
 
+    /**
+     * Gets the class name of {@link com.hazelcast.cache.CacheMergePolicy}
+     * implementation of this cache config.
+     *
+     * @return the class name of {@link com.hazelcast.cache.CacheMergePolicy}
+     *         implementation of this cache config
+     */
+    public String getMergePolicy() {
+        return mergePolicy;
```

Comment:
can you also update `ConfigXmlGenerator.java` class, since man-center relies that class's xml generation.